environments/kubernetes/kube-thanos.libsonnet

```diff
@@ -17,6 +19,30 @@ local rolebinding = k.rbac.v1.roleBinding;
 (import 'thanos-receive-controller/thanos-receive-controller.libsonnet') +
 (import '../../components/thanos-querier-cache.libsonnet') +
 {
+  local jaegerAgent =
```

Comment:
Not super important here, but it seems that it would be very useful to have a mixin (non-monitoring-mixin, just a language paradigm mixin) for this, as this will come up a lot I imagine. Thinking declarative sidecar injections.

Reply:
For a start I can at least refactor it into another package/directory, so that other _could_ grab that folder only.

Reply:
Sounds good!